editor/gui/editor_file_dialog.cpp

```diff
@@ -1712,6 +1713,22 @@ bool EditorFileDialog::are_previews_enabled() {
 	return previews_enabled;
 }
 
+void EditorFileDialog::add_side_menu(Control *p_menu, const String &p_title) {
+	// HSplitContainer has 3 children at maximum capacity, 1 of them is the SplitContainerDragger.
```

Comment:
Should be 2, I think?

Reply:
HSplitContainer has 3 children at maximum capacity, 1 of them is the SplitContainerDragger.

* User-facing child A.
* User-facing child B.
* SplitContainerDragger.

Reply:
Ah right, I forgot about the dragger being a dedicated control.